### ISSUE TITLE
check can login but cant read the message

### ISSUE THREAD

**Issue by hagenbauer** [ThomasLeister/check_xmpp#3]
Env
- Rasbian 8.0
- go version go1.3.3 linux/arm
- xmpp ejabberd-17.12

I create a special account on my server for monitoring tasks. I can login into this accout with the conversatio app on android. 

when I try to execute a check my conversation app on android is killed and the check is running into a timeout. 
Any ideas?

./check_xmpp -timeout 15s -userid monitoring@caserio.de -password secret -debug
2018/02/17 10:48:01 XMPP check starting ... 
2018/02/17 10:48:01 Remote host: chat.caserio.de.
2018/02/17 10:48:01 Connecting to XMPP host chat.caserio.de. ...
2018/02/17 10:48:02 Authentication successful
2018/02/17 10:48:02 XMPP stream is established
2018/02/17 10:48:02 Sending message ...
2018/02/17 10:48:02 Waiting for message to arrive ...
CRITICAL - Timeout

**Comment by ThomasLeister** [comment]
Does the XMPP server work in general? (Can we be sure this is a check_xmpp issue?) Your server seems to crash if either 

- any message is sent ... or
- a user sends at message to himself/herself (this is how the check works)

**Comment by hagenbauer** [comment]
yes. the xmpp server is working. my server is not crashing. just the android app

**Comment by ThomasLeister** [comment]
I have no idea what causes this :( Maybe you can try and use a XMPP account that is not connected to any other client?

**Comment by hagenbauer** [comment]
yes. test that already, no luck
